services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/core/Utils.java

```diff
@@ -117,4 +119,13 @@ public static String cleanAttributeName(String key) {
         return sortKeyValue.map(attributeValue -> Key.create(partitionKeyValue, attributeValue))
                            .orElseGet(() -> Key.create(partitionKeyValue));
     }
+
+    public static <T> List<T> getListIfExist(List<Supplier<T>> itemSupplierList) {
```

Comment:
Now it's generic and global we should rename this to be a little more specific to what it does, which is call each supplier and make a list of the results.

Reply:
yes, definitely. Updated.